extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -164,6 +166,18 @@ public void build(RecorderContext recorder, HibernateOrmTemplate template,
         JpaJandexScavenger scavenger = new JpaJandexScavenger(reflectiveClass, descriptors, compositeIndex, nonJpaModelClasses);
         final JpaEntitiesBuildItem domainObjects = scavenger.discoverModelAndRegisterForReflection();
 
+        // register the strategies of @GenericGenerator for reflection
+        Set<String> strategies = new HashSet<>();
+        for (AnnotationInstance annotation : index.getIndex().getAnnotations(GENERIC_GENERATOR)) {
+            String strategy = annotation.value("strategy").asString();
```

Comment:
I think the `asString()` method could hit an NPE ?

Reply:
I thought it couldn't because the `strategy` field is necesarry for `GenericGenerator`, but I guess we can be defensive, it doesn't hurt :)